### ISSUE TITLE
Training set availability

### ISSUE THREAD

**Issue by redna11** [hugowschneider/longdist.py#1]
Hello there, from the paper relative to this app it says: "To train the models, 18,200 PCTs and 18,200 lncRNAs were used from GRCh37, and 21,018 PCTs and 21,018 lncRNAs fromGRCh38. GRCh37 testing data set included 6,066 PCTs and 6,066 lncRNAs, while the GRCh38 testing data set contained 7,006 PCTs and 7,006 lncRNAs." 
I see that only the test set is included here,
Any chance to upload the training set as well so that it is possible to replicate the results from the research paper? Many thanks!

**Comment by hugowschneider** [comment]
Hi,

I used data available in ENSEMBL (https://useast.ensembl.org/index.html). Unfortunately I don't have the exact datasets anymore.

But for reproducing the experiment you can download the datasets from ENSEMBL. The lncRNAs datasets are all sequences in the ncRNA fasta files which the biotype matches lncRNAs. The biotypes are listed in this page:
https://useast.ensembl.org/info/genome/genebuild/biotypes.html

The human datasets where downloaded as following:

GRCh38:
PCT: 
 ftp://ftp.ensembl.org/pub/release-94/fasta/homo_sapiens/cds
NcRNAs:
 ftp://ftp.ensembl.org/pub/release-94/fasta/homo_sapiens/ncrna

GRCh37:
PCT: 
ftp://ftp.ensembl.org/pub/grch37/update/fasta/homo_sapiens/cds
NcRNAs:
ftp://ftp.ensembl.org/pub/grch37/update/fasta/homo_sapiens/ncrna

The amount of PCTs is of course bigger than lncRNAs. The PCTs were randomly chosen to fit the amount of lncRNAS. (the script already randomly choose the sequences, so for training and testing the amount of negative samples and positive samples are the same)

For other species, the process was the same.

I hope it helps

**Comment by redna11** [comment]
that helps thanks
